documents/Libraries/stdlib/sx-glsl/mx_gamma24_to_lin_rec709_color3.glsl

```diff
@@ -0,0 +1,5 @@
+void mx_gamma24_to_lin_rec709_color3(vec3 _in, out vec3 result)
```

Comment:
One thought here:

Since this code represents a pure transfer function, it can technically be used with any color gamut (e.g. rec2020, acescg), and isn't specific to the rec709 gamut.  Perhaps it could simply be named "mx_gamma24_to_linear_color3"?